### PR TITLE
Describe current behavior on the cache interface

### PR DESCRIPTION
**Pull request by WyriHaximus**
After a discussion with @jsor and @clue I've crafted this PR to only describe the current behavior.

### CODE REVIEW

File: src/CacheInterface.php [hunk 1]
```diff
@@ -2,12 +2,33 @@
 
 namespace React\Cache;
 
+use React\Promise\PromiseInterface;
+
 interface CacheInterface
 {
-    // @return React\Promise\PromiseInterface
+    /**
+     * Retrieve an item from cache, resolves with its value on
```
Comment (jsor) [inline src/CacheInterface.php]: Retrieve an item from **the** cache

File: src/CacheInterface.php [hunk 2]
```diff
@@ -2,12 +2,33 @@
...
+     * success or rejects on when no item can be found.
```
Comment (jsor) [inline src/CacheInterface.php]: ~on~

File: src/CacheInterface.php [hunk 3]
```diff
@@ -2,12 +2,33 @@
...
+     *
+     * @param string $key
+     * @return PromiseInterface
+     */
     public function get($key);
 
+    /**
+     * Stores an item in the cache.
```
Comment (jsor) [inline src/CacheInterface.php]: Store

**Comment by WyriHaximus** [comment]
@jsor cheers, fixed the issues you pointed out :+1:

**Comment by WyriHaximus** [comment]
@clue sure thing, once @jsor approves as well I'll squash and merge

**Pull request merged**